app/controllers/api/v3/analytics/user_analytics_controller.rb

```diff
@@ -10,14 +10,17 @@ def show
     @period = Period.month(Date.current)
     @user_analytics = UserAnalyticsPresenter.new(current_facility)
     @service = Reports::FacilityProgressService.new(current_facility, @period)
+
     @total_follow_ups_dimension = Reports::FacilityProgressDimension.new(:follow_ups, diagnosis: :all, gender: :all)
     @total_registrations_dimension = Reports::FacilityProgressDimension.new(:registrations, diagnosis: :all, gender: :all)
     @total_follow_ups = Reports::MonthlyProgressComponent.new(@total_follow_ups_dimension, service: @service).total_count
     @total_registrations = Reports::MonthlyProgressComponent.new(@total_registrations_dimension, service: @service).total_count
-    if Flipper.enabled?(:new_progress_tab)
-      @period_reports_data = Reports::ReportsFakeFacilityProgressService.new(@current_facility.name).period_reports
-      @hypertension_reports_data = Reports::ReportsFakeFacilityProgressService.new(@current_facility.name).hypertension_reports
-      @diabetes_reports_data = Reports::ReportsFakeFacilityProgressService.new(@current_facility.name).diabetes_reports
+
+    @drug_stocks = DrugStock.latest_for_facilities_grouped_by_protocol_drug(current_facility, @for_end_of_month)
+    unless @drug_stocks.empty?
+      @query = DrugStocksQuery.new(facilities: [current_facility],
```

Comment:
Let's call this `@drug_stocks_query`. There's a lot going on in this page so`@query` wouldn't make it very clear what it is.